### PR TITLE
[LDAP] Add protected IMPORT role to ldap

### PR DESCRIPTION
**Pull request by f-necas**
Add import role to LDAP, to be able to allow just part of users to use the import functionnality (datafeeder app).

Header PR : 
- https://github.com/georchestra/header/pull/22
⚠️ 🛑  This PR should not be merged until versioning of header's cdn is ready. As it will impact actuals configurations.

Datadir PR : 
- https://github.com/georchestra/datadir/pull/390

### CODE REVIEW

File: ldap/docker-root/georchestra.ldif [hunk 1]
```diff
@@ -260,6 +260,16 @@ description: This role allows to use the emailProxy webservice from the console
 member: uid=testadmin,ou=users,dc=georchestra,dc=org
 georchestraObjectIdentifier: 2b5150f7-6451-4bd7-8ecb-b44ffa39fa08
 
+dn: cn=IMPORT,ou=roles,dc=georchestra,dc=org
+objectClass: georchestraRole
+objectClass: groupOfMembers
+objectClass: top
+cn: IMPORT
+description: This role grants access to import functionnality through datafeeder application.
```
Comment (fvanderbiest) [inline ldap/docker-root/georchestra.ldif:268]: functionality

Comment (fvanderbiest) [reply ldap/docker-root/georchestra.ldif:268]: better: "This role allows users to publish data with the importer application"

**Review by emmdurin — APPROVED** [review]

**Comment by fvanderbiest** [comment]
LGTM, thanks !

**Comment by f-necas** [comment]
Don't forget to add it angularjs ... https://github.com/georchestra/georchestra/commit/888ac936b47914ba18878e8ce51733ba20eaa506

**Pull request merged**